gemfiles/.bundle/config

```diff
@@ -0,0 +1,2 @@
+---
+BUNDLE_RETRY: "1"
```

Comment:
We should add `gemfiles/.bundle/` to the `.gitignore` so we don't commit this file